### DISCUSSION TITLE
Beginner Questions

### DISCUSSION THREAD

**Discussion by NotReallyNotNotMe** [YOURLS/YOURLS#3254] [Q&A] [answered]
Followed install instructions. BUT do not understand / know and can not find a beginner's guide.
1. What url do i enter to get a shortened url?
2. When I go to the site where installed, I can see a list of folders (see attached). When I click on admin, I get a blank page. I assumed this link is for site administration but must be wrong?
3. Unable to locate any error logs that might be helpful.
/R
![entry page](https://user-images.githubusercontent.com/33203282/158690613-7c3bdb8e-165f-45d0-a9dc-af3b0f3b1f7d.png)

**Comment by dgw** [comment, marked as answer]
Your server isn't processing PHP files. Look at [the wiki pages](https://github.com/YOURLS/YOURLS/wiki/) for more details.

More streamlined documentation is in progress but not public _quite_ yet. 🙂

**Comment by NotReallyNotNotMe** [comment]
DGW,

Well, I've been to wiki pages and that didn't help with processing PHP
files. Since you say PHP, I'll re-look into config.php to see if I got
something wrong. Otherwise, I'm still lost.

My yourls is on bronord.com/youruls - is that the correct route to creating
a shortened link when things are working? I found zero about that basic /
first step!

Thanks for the feed back.

/Roger
linux user, etc.

On Wed, Mar 16, 2022 at 6:00 PM dgw ***@***.***> wrote:

> Your server isn't processing PHP files. Look at the wiki pages
> <https://github.com/YOURLS/YOURLS/wiki/> for more details.
>
> More streamlined documentation is in progress but not public *quite* yet.
> 🙂
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/YOURLS/YOURLS/discussions/3254#discussioncomment-2376228>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH5KIUQJAEX64D2KMNEOSELVAJKV7ANCNFSM5Q5BBFHQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Roger

**Reply by dgw** [reply]
FYI, you mistyped the link to your install. The correct page shows it is served by LiteSpeed Web Server, which is pretty uncommon among YOURLS users. Only one discussion (#3165) has mentioned it since we enabled the discussion feature, and before that the last reference is in a very old issue from before YOURLS even moved to GitHub.

At any rate, the sample configuration for Apache, Nginx, etc. from our documentation likely won't work for LiteSpeed, as it obviously uses its own configuration methods. If you are using a LiteSpeed server provided by a managed hosting provider, consult their help center. If this is your own server, I'd advise you to start with LiteSpeed's own documentation on [using it with PHP](https://docs.litespeedtech.com/extapp/php/getting_started/).

**Reply by NotReallyNotNotMe** [reply]
After numerous tries at fixing config.php, I managed to login but am now having pword problems. Will work on that - looking for pword reset, etc. Thanks.

**Reply by dgw** [reply]
I see that PHP is working in the `admin` section. Didn't actually test that before, so I apologize if my seeing the directory listing at `/yourls` and jumping right to docs about making PHP work put you on a wild goose chase!

There is no password reset function, as such. You just [edit the `config.php` file](https://github.com/YOURLS/YOURLS/wiki/Username-Passwords).
